index.js

```diff
@@ -1,6 +1,86 @@
 function loader (mcVersion) {
+  const Item = require('prismarine-item')(mcVersion)
+  const Window = require('./lib/Window')(Item)
+
+  let windows
+  if (mcVersion.startsWith('1.14')) {
+    // https://wiki.vg/Inventory
+    windows = {
+      'minecraft:inventory': { type: -1, inventory: { start: 9, end: 44 }, slots: 46, craft: 0, requireConfirmation: true },
```

Comment:
very nice, would be nice if we could put this in mcdata (related to current windows.json I guess ?)

Reply:
yes, is the current windows.json actually used anywhere? Can we replace it or do we merge or create another json?